ios/Classes/MapViewPlugin.m

```diff
@@ -31,6 +31,8 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
         NSDictionary *mapOptions = args[@"mapOptions"];
         NSDictionary *cameraDict = mapOptions[@"cameraPosition"];
         self.mapTitle = mapOptions[@"title"];
+
+        self.mapViewType = [mapOptions[@"mapViewType"] intValue];
```

Comment:
current mapViewType could be null or 0 because it's not a required param for MapOptions. You should make sure this is a valid enum value before setting it.